src/python_pachyderm/experimental/mixin/pfs.py

```diff
@@ -988,7 +1160,10 @@ def walk_file(
         )
 
     def glob_file(
-        self, commit: Union[tuple, dict, Commit, pfs_proto.Commit], pattern: str
+        self,
+        commit: Union[tuple, dict, Commit, pfs_proto.Commit],
+        pattern: str,
+        path_range: pfs_proto.PathRange = None,
```

Comment:
param missing docstring

Reply:
This was actually meant to be internal. I removed it.